### PR TITLE
Add LG Account

### PR DESCRIPTION
**Pull request by SteffenGivard**
Adds [LG Account](https://www.lgaccount.com) as easy to delete.

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -10530,6 +10530,17 @@
...
     },
 
+    {
+        "name": "LG Account",
+        "url": "https://www.lgaccount.com",
+        "difficulty": "easy",
+        "notes": "[Log in](https://dk.lgaccount.com/member/delete_account_introduce), open the account administration tab, and press 'Delete Account'. Note that you might need to repeat going to the adminstration tab and pressing delete a couple times depending on the services you've got connected to LG Account. Once there's no more to delete, you should be logged out automatically (for good measure, verify you cannot login again).",
+        "domains": [
+            "lgaccount.com",
+            "www.lgaccount.com"
```
Comment (Kefaku) [inline _data/sites.json:10540]: The `www` part doesn't need to be included, it is automatically included with `lgaccount.com`.
That beeing said, variants like `us.lgaccount.com` and `dk.lgaccount.com` could be added here.

File: _data/sites.json [hunk 2]
```diff
@@ -10530,6 +10530,17 @@
         ]
     },
 
+    {
+        "name": "LG Account",
+        "url": "https://www.lgaccount.com",
+        "difficulty": "easy",
+        "notes": "[Log in](https://dk.lgaccount.com/member/delete_account_introduce), open the account administration tab, and press 'Delete Account'. Note that you might need to repeat going to the adminstration tab and pressing delete a couple times depending on the services you've got connected to LG Account. Once there's no more to delete, you should be logged out automatically (for good measure, verify you cannot login again).",
```
Comment (Kefaku) [inline _data/sites.json:10537]: The danish account deletion page shouldn't be the default in my opinion. Most users are likely not to speak danish.

**Review by Kefaku — CHANGES_REQUESTED** [review]

File: _data/sites.json (hunk 2 above)
Comment (SteffenGivard) [reply _data/sites.json:10537]: Completely agree :)

**Review by Kefaku — APPROVED** [review]

File: _data/sites.json [hunk 3]
```diff
@@ -10530,6 +10530,161 @@
         ]
     },
 
+    {
+        "name": "LG Account",
+        "url": "https://www.lgaccount.com",
+        "difficulty": "easy",
+        "notes": "Log in, open the account administration tab, and press 'Delete Account'. Note that you might need to repeat going to the adminstration tab and pressing delete a couple times depending on the services you've got connected to LG Account. Once there's no more to delete, you should be logged out automatically (for good measure, verify you cannot login again).",
```
Comment (tupaschoal) [inline _data/sites.json:10537]:
```suggestion
        "notes": "Log in, open the account administration tab, and press 'Delete Account'. Note that you might need to repeat going to the administration tab and pressing delete a couple times depending on the services you've got connected to LG Account. Once there's no more to delete, you should be logged out automatically (for good measure, verify you cannot login again).",
```

**Review by tupaschoal — APPROVED** [review]

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**